src/domain/callout/AspectCallout.tsx

```diff
@@ -12,24 +18,128 @@ interface AspectCalloutProps extends OptionalCoreEntityIds {
 }
 
 const AspectCallout = ({ callout, loading, hubNameId, challengeNameId, opportunityNameId }: AspectCalloutProps) => {
+  // Dialog handling
+  const [aspectDialogOpen, setAspectDialogOpen] = useState(false);
+  const handleCreateDialogOpened = () => setAspectDialogOpen(true);
+  const handleCreateDialogClosed = () => setAspectDialogOpen(false);
+
+  // Create aspects
+  const handleError = useApolloErrorHandler();
+  const { canCreateAspects, calloutId, subscriptionEnabled } = useAspectsData({
```

Comment:
`canCreateAspects` is not fetched from the correct place. It must be from the callout.